src/Symfony/Component/Form/Extension/Core/DataAccessor/PropertyPathAccessor.php

```diff
@@ -30,10 +30,12 @@
 class PropertyPathAccessor implements DataAccessorInterface
 {
     private $propertyAccessor;
+    private $fallbackCallbackAccessorForCustomGetter;
 
     public function __construct(?PropertyAccessorInterface $propertyAccessor = null)
     {
         $this->propertyAccessor = $propertyAccessor ?? PropertyAccess::createPropertyAccessor();
+        $this->fallbackCallbackAccessorForCustomGetter = new CallbackAccessor();
```

Comment:
This could be simpler, but it's limited to falling back on `CallbackAccessor` only, when it could be any `DataAccessorInterface` adapter.

See https://github.com/symfony/symfony/pull/54723 for a more generic solution.